src/modules/DisplayTodo.js

```diff
@@ -1,59 +1,109 @@
-    editField.value = description;
-    editField.className = 'edit-field';
-
-    removeBtn.addEventListener('click', () => {
-      removeTodo(index);
-    });
+import RemoveList from './RemoveTodo.js';
+// eslint-disable-next-line import/no-cycle
+import UpdateList from './UpdateTodo.js';
 
-    editBtn.addEventListener('click', () => {
-      todoParagraph.style.display = 'none';
-      checkBox.style.display = 'none';
-      removeBtn.style.display = 'block';
-      editBtn.style.display = 'none';
-      editField.style.display = 'block';
-      editField.focus();
-    });
+class ShowList {
+  showList = () => {
+    let listArr = [];
+    const removeListData = new RemoveList();
+    const completedTask = new CompletedList();
+    const updateData = new UpdateList();
+
+    // Check localStorage
+    if (localStorage.getItem('list') === null) {
+      localStorage.setItem('list', JSON.stringify(listArr));
+    } else {
+      const listArrStr = localStorage.getItem('list');
+      listArr = JSON.parse(listArrStr);
+    }
+
+    // Check if array is empty then show this message
+    if (listArr.length === 0 || localStorage.getItem('list') === null) {
+      const item = document.querySelector('#items');
+      const label = document.createElement('label');
+      label.htmlFor = 'description';
+      label.innerHTML = 'No Todo For Today';
+      label.className = 'not-found';
+      clearCompletedTasks.style.display = 'none';
+      item.appendChild(label);
+    }
+
+    // Fetch all the data from array
+    listArr.map((data) => {
+      const { index, description, completed } = data;
+      const div = document.createElement('div');
+      div.className = 'inner-data';
+      const innerDiv = document.createElement('div');
+      const label = document.createElement('label');
+      label.htmlFor = `description-${index}`;
+      label.setAttribute('id', index);
+      label.innerHTML = `${description}`;
+      label.className = 'description';
+      const checkbox = document.createElement('input');
+      checkbox.type = 'checkbox';
+      checkbox.setAttribute('id', `description-${index}`);
+      const inputField = document.createElement('input');
+      inputField.type = 'text';
+      inputField.value = description;
+      inputField.style.display = 'none';
+      inputField.className = 'update-field';
+      const removeButton = document.createElement('i');
+      removeButton.className = 'fa fa-trash-alt';
+      const editButton = document.createElement('i');
+      editButton.className = 'fa fa-ellipsis-v';
+      const hr = document.createElement('hr');
+
+      // Remove item from the list
+      removeButton.addEventListener('click', () => {
+        removeListData.removeList(index);
+      });
 
-    editField.addEventListener('keydown', (e) => {
-      if (e.key === 'Enter') {
-        updateTodo(index, editField.value);
+      // Update completed tasks list
+      checkbox.addEventListener('click', () => {
```

Comment:
- [ ] I would recommend and as per requirements, It should be `change` event listener instead of `click` on the checkbox. 

As basically click means clicking the checkbox like any other element, and `change` means that the value of the checkbox has been changed. Here we are changing the value that's why `change` event listener is recommended.